translations/default/ModulesBlockreassuranceAdmin.xlf

```diff
@@ -151,15 +151,6 @@ Comment: Confirm uninstall</note>
       </trans-unit>
     </body>
   </file>
-  <file original="modules/blockreassurance/views/templates/admin/tabs/content/config_elements/cms.tpl" source-language="en-US" target-language="en" datatype="plaintext">
```

Comment:
You should modify absolutely NO translation files, you can revert the changes on all those files